modules/ROOT/pages/servers-about.adoc

```diff
@@ -1,11 +1,12 @@
-= Servers (Hybrid)
+= Servers
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
 image:logo-cloud-disabled.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
 image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", title="Hybrid Deployment"]
-image:logo-server-active.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
+image:logo-server-active.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
```

Comment:
Since this is the first occurrence in the topic, should PCE be spelled out?

Reply:
Kim didn't mark it when she reviewed similar.